test/integration/helpers/inspec/helpers_spec.rb

```diff
@@ -1,7 +1,3 @@
-require 'serverspec'
-
-set :backend, :exec
-
```

Comment:
Can you please convert this to our normal symlinked helpers script and add it for the suites that need it?